wiki/People/The_Team/Nomination_Assessment_Team/SEV_rating/en.md

```diff
@@ -0,0 +1,161 @@
+# SEV rating
+
+SEV is a measurement system used by the [Nomination Assessment Team](/wiki/People/The_Team/Nomination_Assessment_Team) (*NAT*) to rate how relevant a [nomination reset](/wiki/Beatmap_ranking_procedure#nomination-resets) is to the evaluations of the nominating [Beatmap Nominators](/wiki/People/The_Team/Beatmap_Nominators) (*BN*), providing both transparency and consistency. This measurement is split up into two values and displayed as "*Obviousness/Severity*". Both values range from 0 to 2, making it a simple to understand and quick to use system.
```

Comment:
what does sev specifically have to do with transparency

Reply:
transparency to how we roughly view each reset and how they might factor in evals, so people know whether a reset is either minor or concerning, and anything in between